lib/mayaUsdAPI/render.h

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright 2024 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef MAYAUSDAPI_RENDER_H
+#define MAYAUSDAPI_RENDER_H
+
+#include <mayaUsdAPI/api.h>
+
+#include <MaterialXCore/Document.h>
+#include <MaterialXCore/Element.h>
+#include <MaterialXCore/Node.h>
+#include <MaterialXFormat/File.h>
```

Comment:
Do you think its okay to include MaterialX headers here (in the .h) or should I try and forward declare everything? We don't do that for pxr includes, so I figured it was okay here. MaterialX comes from the USD artifact. Anytime we update USD there could be binary API changes coming from that update.